### PR TITLE
PL/FE-71 Lock pool as an admin

### PR DESCRIPTION
**Pull request by l-polina**
**Description:**

Switch pool lock and pool lock status table have been created.

**Screenshots:**

Add some screenshots to illustrate your work.
image

### CODE REVIEW

File: src/components/AdminPanel/AdminPanel.jsx [hunk 1]
```diff
@@ -1,10 +1,12 @@
...
-import SwitchDeposit from './Switch/SwitchDeposit';
+import SwitchAnnotation from './SwitchAnnotation/SwitchAnnotation';
+import SwitchPauseOperation from './SwitchPauseOperation/SwitchPauseOperation';
 
-function AdminPanel() {
+function AdminPanel({ account }) {
 	return (
 		<div>
-			<SwitchDeposit />
+			<SwitchPauseOperation account={account} />
```
Comment (DenisRomanovsky) [inline src/components/AdminPanel/AdminPanel.jsx:8]: I'd suggest to rename the components a bit to make the underlying logic clearer. Example: PoolOperationsLocker or PoolOperationsSwitch and PoolOperationsLockStatuses/PoolOperationsStatuses.

File: src/components/AdminPanel/AdminPanel.jsx [hunk 2]
```diff
@@ -1,10 +1,12 @@
 import React from 'react';
-import SwitchDeposit from './Switch/SwitchDeposit';
+import SwitchAnnotation from './SwitchAnnotation/SwitchAnnotation';
+import SwitchPauseOperation from './SwitchPauseOperation/SwitchPauseOperation';
```
Comment (DenisRomanovsky) [inline src/components/AdminPanel/AdminPanel.jsx:3]: Let's move admin-related components into admin folder. This'll make a nice structure (as an example:
admin
 -- OperationsLocks
 -- OperationsStatuses
 
The context of a component becomes clear from the folder it lies in.

File: src/components/AdminPanel/SwitchPauseOperation/SwitchPauseOperation.jsx [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+		setInvalid(!(asset && operation && account));
+	};
+
+	const assets = UNDERLYING_ASSETS_TYPES.map((currency) => ({
+		key: currency,
+		text: currency,
+		value: currency,
+	}));
+
+	const operations = OPERATIONS.map((action) => ({
```
Comment (DenisRomanovsky) [inline src/components/AdminPanel/SwitchPauseOperation/SwitchPauseOperation.jsx:30]: OPERATIONS is a bit vague name. What do ou think about POOL_OPERATIONS?

**Comment by DenisRomanovsky** [comment]
Looks great in general.

File: src/components/AdminPanel/AdminPanel.jsx (hunk 1 above)
Comment (l-polina) [reply src/components/AdminPanel/AdminPanel.jsx:8]: > I'd suggest to rename the components a bit to make the underlying logic clearer. Example: PoolOperationsLocker or PoolOperationsSwitch and PoolOperationsLockStatuses/PoolOperationsStatuses.

The components have been renamed.

File: src/components/AdminPanel/SwitchPauseOperation/SwitchPauseOperation.jsx (hunk 3 above)
Comment (l-polina) [reply src/components/AdminPanel/SwitchPauseOperation/SwitchPauseOperation.jsx]: > OPERATIONS is a bit vague name. What do ou think about POOL_OPERATIONS?


The name has been changed.

**Review by DenisRomanovsky — APPROVED** [review]

**Pull request merged**